scripts/dev/upstream_merge/repos.conf

```diff
@@ -1,13 +1,14 @@
-sources/meta-openembedded    https://git.openembedded.org/meta-openembedded        hardknott             nilrt/master/hardknott
-sources/meta-selinux         https://git.yoctoproject.org/meta-selinux             hardknott             nilrt/master/hardknott
-sources/openembedded-core    https://git.openembedded.org/openembedded-core        hardknott             nilrt/master/hardknott
-sources/meta-virtualization  https://git.yoctoproject.org/meta-virtualization      hardknott             nilrt/master/hardknott
-sources/meta-security        https://git.yoctoproject.org/meta-security            hardknott             nilrt/master/hardknott
-sources/meta-sdr             https://github.com/balister/meta-sdr                  hardknott             nilrt/master/hardknott
-sources/meta-qt5             https://github.com/meta-qt5/meta-qt5                  hardknott             nilrt/master/hardknott
-sources/meta-qt5-extra       https://github.com/schnitzeltony/meta-qt5-extra       hardknott             nilrt/master/hardknott
-sources/meta-rauc            https://github.com/rauc/meta-rauc                     hardknott             nilrt/master/hardknott
-sources/pyrex                https://github.com/garmin/pyrex                       master                nilrt/master/hardknott
+sources/bitbake              https://git.openembedded.org/bitbake                  2.0                   nilrt/master/kirkstone
+sources/meta-cloud-services  https://git.yoctoproject.org/meta-cloud-services      kirkstone             nilrt/master/kirkstone
+sources/meta-openembedded    https://git.openembedded.org/meta-openembedded        kirkstone             nilrt/master/kirkstone
+sources/meta-selinux         https://git.yoctoproject.org/meta-selinux             kirkstone             nilrt/master/kirkstone
+sources/openembedded-core    https://git.openembedded.org/openembedded-core        kirkstone             nilrt/master/kirkstone
+sources/meta-virtualization  https://git.yoctoproject.org/meta-virtualization      kirkstone             nilrt/master/kirkstone
+sources/meta-security        https://git.yoctoproject.org/meta-security            kirkstone             nilrt/master/kirkstone
+sources/meta-sdr             https://github.com/balister/meta-sdr                  kirkstone             nilrt/master/kirkstone
+sources/meta-qt5             https://github.com/meta-qt5/meta-qt5                  kirkstone             nilrt/master/kirkstone
+sources/meta-qt5-extra       https://github.com/schnitzeltony/meta-qt5-extra       kirkstone             nilrt/master/kirkstone
+sources/meta-rauc            https://github.com/rauc/meta-rauc                     kirkstone             nilrt/master/kirkstone
+sources/pyrex                https://github.com/garmin/pyrex                       master                nilrt/master/kirkstone
+sources/meta-mingw           https://git.yoctoproject.org/meta-mingw               kirkstone             nilrt/master/kirkstone
```

Comment:
We seem to have [removed](https://github.com/ni/nilrt/commit/68a63c866b9de1b4fd23ce36306e7114f29c6945) `meta-mingw` layer from hardknott and [added it back](https://github.com/ni/nilrt/commit/031b41a6a392b03f54e141c902144b21ceed82c0) but we don't have it listed in hardknott's `repos.conf`. Do you mind adding it?

Reply:
Can do

Reply:
https://github.com/ni/nilrt/pull/208

Reply:
The local branch for meta-mingw appears to be wrong according to the upstream merge script. I'll correct it

Reply:
https://github.com/ni/nilrt/pull/209